src/components/[guild]/leaderboard/LeaderboardPointsSelector.tsx

```diff
@@ -30,9 +32,25 @@ const LeaderboardPointsSelector = () => {
   if (pointsRewards.length < 2) {
     if (isAdmin)
       return (
-        <Card borderRadius="xl" flexShrink={0}>
-          <RecalculateLeaderboardButton />
-        </Card>
+        <ButtonGroup>
```

Comment:
This will only be displayed if there are less than 2 point rewards, which is not correct hehe. And while I like this new design, I'd go for consistency here & use the same design for all cases.